test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml

```diff
@@ -68,3 +68,5 @@ user_prefix: 'longevity-large-partitions-200k-pks-4d'
 space_node_threshold: 644245094
 
 stop_test_on_stress_failure: false
+
+run_full_partition_scan: '{"ks_cf": "scylla_bench.test", "interval": 2, "pk_name":"pk", "rows_count": 100000}' # 'ks.cf, interval(min), partition-key name, number-of-rows-per-partition'
```

Comment:
What  is the rows_count used for?

Reply:
its number-of-rows-per-partition and is used for taking a random number out of it in order to filter the ck.